unitystation/unitystation
# Introducing UniCloth prefab

To test UniPrefab:
-Place UniCloth prefab on the scene
-Make sure it's in NetworkManager's spawn list
-Paste any cloth-related string into Hierarchy attribute from Resource/metadata/hier.txt
